org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -235,7 +235,7 @@ GithubOidcSageBionetworksWebMonorepoInfra:
               "Sid": "CloudfrontActions",
               "Effect": "Allow",
               "Action": [ "cloudfront:CreateInvalidation" ],
-              "Resource": ["arn:aws:cloudfront::797640923903:distribution/E2656IE63W1MXI", "arn:aws:cloudfront::797640923903:distribution/E2656IE63W1MXI" ]
+              "Resource": ["arn:aws:cloudfront::797640923903:distribution/E2656IE63W1MXI", "arn:aws:cloudfront::797640923903:distribution/EY52HOUGKDP1F" ]
           }
         ]
       }
```

Comment:
If we wish to deploy when we push to a "production" branch, or create a tag, will we need to change `TemplatingContext.Repositories[*].branch` below? I.O.W, will we only be able to use these AWS creds on deployments triggered by our main branch?